specification/resources/resource-manager/Microsoft.Authorization/stable/2020-09-01/dataPolicyManifests.json

```diff
@@ -0,0 +1,430 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "PolicyClient",
+    "version": "2020-09-01",
+    "description": "To manage and control access to your resources, you can define customized policies and assign them at a scope."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.Authorization/dataPolicyManifests/{policyMode}": {
+      "get": {
+        "tags": [
+          "DataPolicyManifests"
+        ],
+        "operationId": "DataPolicyManifests_GetByPolicyMode",
+        "summary": "Retrieves a data policy manifest.",
+        "description": "This operation retrieves the data policy manifest with the given policy mode.",
+        "x-ms-examples": {
+          "Retrieve a data policy manifest by policy mode": {
+            "$ref": "./examples/getDataPolicyManifest.json"
+          }
+        },
+        "parameters": [
+          {
+            "name": "policyMode",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The policy mode of the data policy manifest to get."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns information about a data policy manifest.",
+            "schema": {
+              "$ref": "#/definitions/DataPolicyManifest"
+            }
+          },
+          "404": {
+            "description": "Not found - the data policy manifest doesn't exist."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Authorization/dataPolicyManifests": {
```

Comment:
>dataPolicyManifests [](start = 40, length = 19)

Does this list get large enough to need paging. If there's any chance it will require paging in the future, you should declare this API as pageable just to avoid breaking changes in the future. See [https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html#oapi009-always-add-x-ms-pageable-to-list-calls](https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html#oapi009-always-add-x-ms-pageable-to-list-calls) for details.

Reply:
@feruilob - lets add x-ms-pageable with nextLinkName as null

Reply:
Add it with nextLink set to "nextLink". It's ok if the service never returns that property. It's as if you always returned only one page.